frontend/src/core/register/register.component.ts

```diff
@@ -1,15 +1,51 @@
 import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators, AbstractControl } from '@angular/forms';
+
+function passwordMatcher( c: AbstractControl): { [key: string]: boolean} | null {
```

Comment:
passwordMacher funkcję przenieś z tego miejsca do jakiegoś innego.
Np w shared zrób folder reusable-fuctions czy jakos tak,
potem wrzuć to tam poprzedzając fuction export
POzniej w compoencie zimportuj funkcję i uzyj :)
Będziesz jej używać jeszcze w edycji profilu itd, wiec powinna byc wydzielona